### PR TITLE
Fix ReloadRequest in protocol found while updating to use session in tests

### PR DESCRIPTION
**Pull request by sheetalkamat**
This came up when i was trying to work with projects management and updating tests to use session instead of project service

### CODE REVIEW

File: src/testRunner/unittests/tsserver/projectErrors.ts [hunk 1]
```diff
@@ -583,8 +583,7 @@ describe("unittests:: tsserver:: projectErrors:: dont include overwrite emit err
         const session = new TestSession(host);
         openFilesForSession([f1], session);
 
-        const projectService = session.getProjectService();
-        const projectFileName = projectService.inferredProjects[0].getProjectName();
+        const projectFileName = session.getProjectService().inferredProjects[0].getProjectName();
```
Comment (andrewbranch) [inline src/testRunner/unittests/tsserver/projectErrors.ts:586]: So is `projectService` not safe to reuse between requests now?

**Review by andrewbranch — APPROVED** [review]

File: src/testRunner/unittests/tsserver/projectErrors.ts (hunk 1 above)
Comment (sheetalkamat) [reply src/testRunner/unittests/tsserver/projectErrors.ts:586]: It is but I changed this so it’s easier to track any other usages

**Review by jakebailey — APPROVED** [review]
Changing the protocol is _theoretically_ a breaking change, but isn't clear anyone actually has called this API? https://github.com/search?q=%2F%5Cbts%5C.%5Ba-zA-Z.%5D*ReloadProjects%5Cb%2F&type=code

**Comment by sheetalkamat** [comment]
@jakebailey anyways the api was incorrect for anyone to use ..

**Pull request merged**